packages/office-ui-fabric-react/src/components/Divider/VerticalDivider.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { getDividerClassNames } from './VerticalDivider.classNames';
+import { IVerticalDividerProps } from './VerticalDivider.Props';
```

Comment:
I don't really understand the backstory here, but why not have divider with a vertical boolean flag? That way you can render either way.

Reply:
A. I'd have to implement horizontal now if there was a boolean
B. The styles won't be really shared between the two, so I don't see a ton of benefit

Reply:
Slightly different question; do we really need a line component? Is border or even just....span? not enough? 

And why have 2 spans to render a single line? All you need is a thing that can take up 1px of width and grow 100% of the height, no?

Reply:
We need it to be center aligned with a height of less than 100%. See split button